packages/buttons/README.md

```diff
@@ -0,0 +1,32 @@
+# @cfpb/buttons [![Build Status](https://img.shields.io/travis/cfpb/design-system.svg)](https://travis-ci.org/cfpb/design-system) [![npm](https://img.shields.io/npm/v/@cfpb/buttons.svg?style=flat-square)](https://www.npmjs.com/package/@cfpb/buttons)
+
+Design System buttons
+
+This component can be used by itself, but it was made for Design System,
+a front end framework developed at the
+[Consumer Financial Protection Bureau](https://consumerfinance.gov).
+
+## How to use this component
+
+Detailed instructions can be found at the Design System
+[documentation site](https://cfpb.github.io/design-system/).
+
+> NOTE: Be sure to run the Less files through
```

Comment:
I feel like this note in all the readmes is superfluous. I presume this is for when the components are imported and their source less files are compiled in a project that may have a build system that doesn't incorporate autoprefixer. Let's either move this to one note in the detailed instructions area or remove this entirely (since people could do all kinds of config things that cause issues if they're not careful, which really isn't our bailiwick).